client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -1216,7 +1216,7 @@ static Request getMigrationAssistance(IndexUpgradeInfoRequest indexUpgradeInfoRe
         return request;
     }
 
-    static HttpEntity createEntity(ToXContent toXContent, XContentType xContentType) throws IOException {
+    public static HttpEntity createEntity(ToXContent toXContent, XContentType xContentType) throws IOException {
```

Comment:
I think that if we make all these methods public they should also have javadocs. I think it's a good move that they become public, users could potentially use them without using the high-level client to fire requests, but they become part of our public API this way and we need to be sure that we want that. Maybe though if you plan to split this into different classes belonging each one to their package, the visibility can go back to what it was after that?

Reply:
some of these method will move in a PR i plan on doing that splits up the request converters, but things like createEntity is shared by all of the clients, so it probably makes sense to add javadoc.